codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/http/ResponseBindingGeneratorTest.kt

```diff
@@ -95,21 +96,28 @@ class ResponseBindingGeneratorTest {
         val testProject = TestWorkspace.testProject(symbolProvider)
         testProject.renderOperation()
         testProject.withModule(symbolProvider.moduleForShape(outputShape)) {
-            unitTest(
```

Comment:
Are we forgoing the `unitTest` style of writing unit tests?

Reply:
At least in my opinion, the `unitTest` function just leads to deeper nesting and doesn't really buy me anything when I have a predetermined test name.